priv/templates/nimble_template/.github/wiki/Icon-Sprite.md

```diff
@@ -0,0 +1,20 @@
+# Icon Sprite
```

Comment:
Should we skip this heading, seems the wiki page title is already there, will this duplicate with that title 🤔

Reply:
Yeah, correct @rosle 👍I will update this

Reply:
updated in https://github.com/nimblehq/elixir-templates/pull/146/commits/5db504f70b06726df8efe19e74b4abd1e173611f